cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -100,52 +100,51 @@ Auto-Discovery Setup is the preferred method to configure Cluster Autoscaler.
-
-Each Auto Scaling Group should be composed of instance types that provide
-approximately equal capacity. For example, ASG "xlarge" could be composed of
-m5a.xlarge, m4.xlarge, m5.xlarge, and m5d.xlarge instance types, because each of
-those provide 4 vCPUs and 16GiB RAM. Separately, ASG "2xlarge" could be
-composed of m5a.2xlarge, m4.2xlarge, m5.2xlarge, and m5d.2xlarge instance
-types, because each of those provide 8 vCPUs and 32GiB RAM.
+Cluster Autoscaler leaves untouched the minimum and maximum values of the discovered Auto
+Scaling Group(s). It only manages the desired value.
 
-Cluster Autoscaler will attempt to determine the CPU, memory, and GPU resources
+Cluster Autoscaler determines the CPU, memory, and GPU resources
 provided by an Auto Scaling Group based on the instance type specified in its
-Launch Configuration or Launch Template. It will also examine any overrides
+Launch Configuration or Launch Template. It also examines any overrides
 provided in an ASG's Mixed Instances Policy. If any such overrides are found,
-only the first instance type found will be used.  See [Using Mixed Instances
+only the first instance type found is used.  See [Using Mixed Instances
 Policies and Spot Instances](#Using-Mixed-Instances-Policies-and-Spot-Instances)
 for details.
 
-From version 1.14, Cluster Autoscaler can also determine the resources provided
+From version 1.14, Cluster Autoscaler can _also_ determine the resources provided
 by each Auto Scaling Group via tags. The tag is of the format
 `k8s.io/cluster-autoscaler/node-template/resources/<resource-name>`.
-`<resource-name>` is the name of the resource, such as `ephemeral-storage`. The
-value of each tag specifies the amount of resource provided. The units are
+Where `<resource-name>` is the name of the resource, such as `ephemeral-storage`. The
+value of such a tag specifies the amount of resource provided. The units are
 identical to the units used in the `resources` field of a Pod specification.
 
-Example tags:
+Example tag:
 
 * `k8s.io/cluster-autoscaler/node-template/resources/ephemeral-storage`: `100G`
 
-You may also provide additional hints to Cluster Autoscaler that the nodes will
-be labeled or tainted when they join the cluster, such as:
+
+You may currently be adding labels to taints to your nodes via kubelet configuration. If you want to provide this information as additional hints to Cluster Autoscaler that the nodes will
+be labeled or tainted when they join the cluster, you may do so by adding these labels/taints as tags to your ASGs, and passing the tag-keys to the `--node-group-auto-discovery` flag.
+
+Example labels/taints ASG tags:
 
 * `k8s.io/cluster-autoscaler/node-template/label/foo`: `bar`
 * `k8s.io/cluster-autoscaler/node-template/taint/dedicated`: `NoSchedule`
 
-**NOTE:** It is your responsibility to ensure such labels and/or taints are
-applied via the node's kubelet configuration at startup.
+which can be passed to `--node-group-auto-discovery` like so:
```

Comment:
Duplicating mentioning the `--node-group-auto-discovery` flag. Maybe cleaner with 
```suggestion
which can be passed to the Cluster Autoscaler like so:
```
?